setting_up_firely_server/deployment/azureMarketplace.rst

```diff
@@ -0,0 +1,65 @@
+.. _deploy_azure_marketplace:
+
+Using Firely Server from Azure Marketplace
+==========================================
+
+Firely Server is available in the Azure Marketplace, which allows you to deploy it easily on Azure Kubernetes Service (AKS) with a single click. 
+This deployment method uses an embedded Helm chart, which simplifies the setup process and provides a consistent configuration experience.
+The Azure Marketplace deployment is ideal for users who want to quickly get started with Firely Server on Azure without needing to manually 
+configure Kubernetes resources.
+
+In order to deploy Firely Server from the Azure Marketplace, follow these steps:
+
+1. Go to the `Azure Marketplace <https://azuremarketplace.microsoft.com/en-us/marketplace/apps?search=Firely&page=1>`_ and search for "Firely Server".
+2. Click on the "Create" button to start the deployment process.
+3. On the first panel, provide the Azure subscription, the resource group and the cluster where Firely Server should be deployed.
+4. On the second pane, provide the parameters for configuring the deployment. 
+5. Click "Review + Create" to review your settings and then click "Create" to deploy Firely Server.
+
+You can monitor the deployment progress in the Azure portal under the "Deployments" section of the selected resource group.
+
+For the most basic deployment, you can simply specify the extension resource name and the license, and leave the other parameters at their default values.
```

Comment:
I think it requires and extra sentence what the "extension resource name" is.

Reply:
As long as it's not hard-coded, should we suggest to use "firely-server-market-place"?

Reply:
Yes... but at the moment, it is not possible. I need to modify the constrain on the UI side because so far the dash is not allowed. I can change, republish and try out, even providing this a default value.